streams/sv.m3u

```diff
@@ -11,6 +11,8 @@ https://cloud2.streaminglivehd.com:1936/8048/8048/playlist.m3u8
 http://45.5.116.178:8000/play/a03p/index.m3u8
 #EXTINF:-1 tvg-id="Canal4HD.sv",Canal 4 (1080p)
```

Comment:
"Canal4SD" and "Canal4HD" doesn't exist on database.

Reply:
Fixed